src/v/security/audit/audit_log_manager.cc

```diff
@@ -541,6 +552,17 @@ ss::future<> audit_log_manager::resume() {
     });
 }
 
+double audit_log_manager::client_percent_full() const {
+    vassert(
+      ss::this_shard_id() == audit_client::shard_id,
+      "Must be called on audit client shard");
+    return _sink->percent_full();
+}
+
+ss::shard_id audit_log_manager::client_shard_id() const {
```

Comment:
Maybe we can make `client_shard_id` a public constexpr variable now that it is expected to be consumed by a public interface

Reply:
As in, move it to `audit_log_manager` and have the module-internal classes refer to that?

Reply:
Oh, nvm. Looks like it's only referenced in manager routines anyway.